### ISSUE TITLE
URL set in feature branch configuration does not work properly

### ISSUE THREAD

**Issue by EmlynK** [codeenigma/deployments#127]
We have the ability to set a URL template in config.ini, like this:

```
[featurebranch]
urltemplate=reponame.branchname.example.com
```

In FeatureBranches.py, we set the `url` variable like so, if the *urltemplate* option is present. But before that, we define the global `url` variable:
https://github.com/codeenigma/deployments/blob/9dbb0854f672bbf7be400e99351f8d6180e13c90/drupal/FeatureBranches.py#L51-L53
https://github.com/codeenigma/deployments/blob/9dbb0854f672bbf7be400e99351f8d6180e13c90/drupal/FeatureBranches.py#L111-L119

However, we do not pass in the correct variable to be used here. We need to pass in the `alias` variable.

Plus, I'm not sure if the URL set in the FeatureBranches.py function will stick as back in fabfile.py, the `url` variable is then generated:
https://github.com/codeenigma/deployments/blob/9dbb0854f672bbf7be400e99351f8d6180e13c90/drupal/fabfile.py#L227-L228

I'll run some tests once I've fixed the variable being passed in issue.

**Comment by EmlynK** [comment]
I've done some testing and this is now working as expected in develop. So once develop is merged into master, this can be resolved.
